include/gl_depth_sim/interfaces/noise_model.h

```diff
@@ -0,0 +1,17 @@
+#ifndef NOISE_MODEL_H
+#define NOISE_MODEL_H
+
+#include <math.h>
```

Comment:
Prefer the C++ equivalent header `<cmath>` over the C include `math.h`.

They define the same things but the cmath includes functions in the `std::` namespace.